### PR TITLE
Bugfix/KAYAKO-14684 XCODE 11 SWIFT4.2

### PR DESCRIPTION
**Pull request by romixo**
JIRA:https://jira.devfactory.com/browse/KAYAKO-14684
RCA: XCode 11 no longer supports Swift 3
Fix: Updated Project to  Swift 4.2

### CODE REVIEW

File: Sources/Messenger/Source/Conversations/ConversationsViewController.swift [hunk 1]
```diff
@@ -127,7 +128,7 @@ class ConversationsViewController: UIViewController, ASTableDelegate, StatefulVi
 }
 
 extension UINavigationController {
-	open override var childViewControllerForStatusBarStyle: UIViewController? {
+    open override var childForStatusBarStyle: UIViewController? {
```
Comment (Yabgu) [inline Sources/Messenger/Source/Conversations/ConversationsViewController.swift]: Formatting broken

File: Sources/Messenger/Source/Conversations/ConversationsViewController.swift [hunk 2]
```diff
@@ -99,7 +100,7 @@ class ConversationsViewController: UIViewController, ASTableDelegate, StatefulVi
 		//: For sticky header
 		self.automaticallyAdjustsScrollViewInsets = true
 		self.title = "Conversations"
-		self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedStringKey.foregroundColor: UIColor.white]
+        self.navigationController?.navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.white]
```
Comment (Yabgu) [inline Sources/Messenger/Source/Conversations/ConversationsViewController.swift]: Formatting broken

File: Sources/Messenger/Source/Conversations/ConversationsViewController.swift (hunk 1 above)
Comment (romixo) [reply Sources/Messenger/Source/Conversations/ConversationsViewController.swift]: Fixed

File: Sources/Messenger/Source/Conversations/ConversationsViewController.swift (hunk 2 above)
Comment (romixo) [reply Sources/Messenger/Source/Conversations/ConversationsViewController.swift]: Fixed

**Pull request merged**